const catchError = console.error;

export default {
  catchError,
};
